tests/rule_based_profiler/parameter_builder/test_regex_pattern_string_parameter_builder.py

```diff
@@ -277,28 +326,65 @@ def test_regex_pattern_string_parameter_builder_bobby_no_match(
     )
 
 
-def test_regex_pattern_string_parameter_builder_alice_single_regex_pattern(
-    alice_columnar_table_single_batch_context,
+@mock.patch("great_expectations.data_context.data_context.DataContext")
+def test_regex_wrong_domain(mock_data_context: mock.MagicMock, batch_fixture: Batch):
+    batch: Batch = batch_fixture
+    mock_data_context.get_batch_list.return_value = [batch]
+    mock_data_context.get_validator_using_batch_list.return_value = Validator(
+        execution_engine=PandasExecutionEngine(), batches=[batch]
+    )
+
+    data_context: DataContext = mock_data_context
+
+    # column : c does not exist
+    metric_domain_kwargs: dict = {"column": "c"}
+    candidate_regexes: List[str] = [r"^\d{1}$"]
+
+    regex_pattern_string_parameter_builder: RegexPatternStringParameterBuilder = (
+        RegexPatternStringParameterBuilder(
+            name="my_regex_pattern_string_parameter_builder",
+            metric_domain_kwargs=metric_domain_kwargs,
+            candidate_regexes=candidate_regexes,
+            batch_list=[batch],
+            data_context=data_context,
+        )
+    )
+
+    parameter_container: ParameterContainer = ParameterContainer(parameter_nodes=None)
+    domain: Domain = Domain(
+        domain_type=MetricDomainTypes.COLUMN, domain_kwargs=metric_domain_kwargs
+    )
+    with pytest.raises(ge_exceptions.ProfilerExecutionError) as e:
+        regex_pattern_string_parameter_builder.build_parameters(
+            parameter_container=parameter_container, domain=domain
+        )
+    assert (
```

Comment:
If you are going to push another commit, can you please add a blank line above `assert`?  Thanks!

Reply:
done!